server/routes/userRoutes.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+
+import users from '../controllers/user';
+import middleware from '../middleware/usersValidations';
```

Comment:
According to the AirBnB style guide, what you are importing should have the same name as the file. Here the file is 'usersValidations.js', you are importing 'middleware'. In the base file, you can change either the name of the object you are exporting  or the file name to be the same.